Jellyfin.Api/Controllers/EnvironmentController.cs

```diff
@@ -154,7 +154,7 @@ public IEnumerable<FileSystemEntryInfo> GetDrives()
         /// <returns>Parent path.</returns>
         [HttpGet("ParentPath")]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        public ActionResult<string?> GetParentPath([FromQuery, BindRequired] string path)
+        public ActionResult<string?> GetParentPath([FromQuery, Required] string path)
```

Comment:
Same as above

Reply:
Just tested, [Required] works with both nullable and non-nullable strings